app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Action.java

```diff
@@ -23,6 +23,15 @@
     @SerializedName("query")
     @Expose
     private String query;
+    @SerializedName("latitude")
+    @Expose
+    private String latitude;
+    @SerializedName("longitude")
+    @Expose
+    private String longitude;
+    @SerializedName("zoom")
+    @Expose
+    private String zoom;
```

Comment:
@mayank408 there is type,latitude,longitude,zoom in action

"actions": [
      {
        "type": "answer",
        "expression": "Singapore is a place with a population of 3547809. Here is a map: https://www.openstreetmap.org/#map=13/1.2896698812440377/103.85006683126556"
      },
      {
        "type": "anchor",
        "link": "https://www.openstreetmap.org/#map=13/1.2896698812440377/103.85006683126556",
        "text": "Link to Openstreetmap: Singapore"
      },
      {
        "type": "map",
        "latitude": "1.2896698812440377",
        "longitude": "103.85006683126556",
        "zoom": "13"
      }
    ]